lib/configuration/configuration.js

```diff
@@ -1,5 +1,8 @@
 const yaml = require('js-yaml')
 const _ = require('lodash')
+const NodeCache = require('node-cache')
```

Comment:
I don't have a strong opinion here on choice of caching modules, but am curious on yours here.

Reply:
This was the most popular and well supported/recently updated cache module that support TTL.